### PR TITLE
Edge case tests added

### PR DESCRIPTION
**Pull request by locacow**

### CODE REVIEW

File: AreaCalc.test.js [hunk 1]
```diff
@@ -2,28 +2,23 @@ import test from 'node:test';
...
-//     assert.throws(() => {
-//         calculateAreaFixed('a',3);
-//     }, {
-//         name: 'Error',
-//         message: 'Length and width must be positive numbers'
-//     });});
-
-
+test('Test throws error for non numerical values a*3', (t) => {
+  assert.throws(() => {
```
Comment (themanagerstale-web) [inline AreaCalc.test.js:18]: why not bla bla?

File: AreaCalc.test.js
Comment (themanagerstale-web) [inline AreaCalc.test.js:1]: another comment
